### ISSUE TITLE
First slide flickering when scrolling backwards.

### ISSUE THREAD

**Issue by michalczaplinski** [FormidableLabs/nuka-carousel#140]
In the below example, the first slide disappears from the screen when scrolling backwards:
![carousel](https://cloud.githubusercontent.com/assets/5417266/18124192/7c8a41f8-6f71-11e6-9957-252671143b8e.gif)

``` javascript
<Carousel
  autoplay={true}
  cellAlign='center'
  slidesToShow={2}
  cellSpacing={20}
  wrapAround={true}
  slideWidth='400px'
  decorators={[]}>
    <img src="http://placehold.it/1000x400/ffffff/c0392b/&text=slide1"/>
    <img src="http://placehold.it/1000x400/ffffff/c0392b/&text=slide2"/>
    <img src="http://placehold.it/1000x400/ffffff/c0392b/&text=slide3"/>
</Carousel>
```

**Comment by guciek27** [comment]
+1 I have actually the same problem with the flickering.

**Comment by serkandurusoy** [comment]
Hmm, does anyone have a workaround for this? I also tried going from a single slide to multiple slides and discovered this :/

**Comment by apkoponen** [comment]
I can confirm this too.

**Comment by nowarkdev** [comment]
Created PR to fix this: #148
